tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraEmailSendMethodContract.scala

```diff
@@ -0,0 +1,350 @@
+package com.linagora.tmail.james.common
+
+import com.linagora.tmail.james.common.EncryptHelper.uploadPublicKey
+import com.linagora.tmail.james.common.LinagoraEmailSendMethodContract.BOB_INBOX_PATH
+import io.netty.handler.codec.http.HttpHeaderNames.ACCEPT
+import io.restassured.RestAssured.{`given`, requestSpecification}
+import io.restassured.http.ContentType.JSON
+import net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson
+import org.apache.http.HttpStatus
+import org.apache.james.GuiceJamesServer
+import org.apache.james.jmap.core.ResponseObject.SESSION_STATE
+import org.apache.james.jmap.http.UserCredential
+import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, ACCOUNT_ID, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.mailbox.model.{MailboxPath, MessageId}
+import org.apache.james.mime4j.dom.Message
+import org.apache.james.modules.MailboxProbeImpl
+import org.apache.james.utils.DataProbeImpl
+import org.junit.jupiter.api.{BeforeEach, Test}
+
+import java.nio.charset.StandardCharsets
+
+object LinagoraEmailSendMethodContract {
+  val MESSAGE: Message = Message.Builder.of
+    .setSubject("test")
+    .setSender(BOB.asString)
+    .setFrom(BOB.asString)
+    .setTo(BOB.asString)
+    .setBody("test mail", StandardCharsets.UTF_8)
+    .build
+
+  val MESSAGE_PREVIEW: String = "test mail"
+
+  val BOB_INBOX_PATH: MailboxPath = MailboxPath.inbox(BOB)
+}
+
+trait LinagoraEmailSendMethodContract {
```

Comment:
Please remember to add the following test:

```
Given BOB have encryption enabled for his account
AND ALICE do not have encryption enabled for her account
WHEN BOB sends an email to ALICE
THEN ALICE can access this email in her INBOX
AND this email is not encrypted
```

Reply:
Yes, I will, This is also the main purpose of Email/send.
anw, Thank you